tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -292,7 +292,7 @@ public void Builds_NetCore31App_UsingNetCore31_DotNetSdkVersion()
                 result.GetDebugInfo());
         }
 
-        [Fact]
+        [Fact(Skip = "1086977")]
```

Comment:
I added a new issue for re-enabling these tests once I have the support for getting settings using `IConfiguration` api